src/ingest_validation_tools/table-schemas/sample.yaml

```diff
@@ -2,283 +2,111 @@ doc_url: TODO
-    enum:
-      - Alcohol-based (ETH)
-      - Aldehyde-based (ALD)
-      - Allprotect tissue reagent (ALL)
-      - Heat stabilization (HST)
-      - Neutral buffered formalin (NBF)
-      - Non-aldehyde based without acetic acid (NAA)
-      - Non-aldehyde with acetic acid (ACA)
-      - Non-buffered formalin (FOR)
-      - Optimum cutting temperature medium (OCT)
-      - Cryopreserved in sucrose-OCT
-      - Other (ZZZ)
-      - PAXgene tissue (PXT)
-      - RNA Later (RNL)
-      - Snap freezing (SNP)
-      - CryoStor
-      - 10% serum-DMSO culture media
-      - UW solution
-      - 4% PFA
-      - MeOH
-      - Liquid Nitrogen (LN)
-      - CMC
-      - dry ice-isopentane
-      - Unknown (XXX)
-      - FFPE
-      - none 
-  description: The final medium in which sample is placed. TODO - Are these the precise values you want? Do you also want to reference a protocol? Does "none" mean there is no long-term preservation? or that it's unknown? FFPE moved here as Sanjay requested. Confirm that this is correct?
-
-  TODO_summation: protocol.io  
-  TODO_nils: Not sure if you want people to choose from list of allowed values or if you want them to link to protocols.io. We need to be clear if we want the full name or abbreviation or a combination (as it is written right now). In the assay metadata we don't use ""none"", we just leave the column blank. What do you prefer?  
-  TODO_type: DOI?  
-  TODO_shin: or PDF  
-  TODO_sanjay: drop down values"
--
-  name: biospecimen_aliquot_temperature_in_preservation_solution_value 
-  constraints:
-    required: True
-  type: number  
-  description: The temperature of the medium during the preservation process. TODO - What does the protocol.io mean here? 
-
-  TODO_summation: protocol.io  
-  TODO_nils: Need value and unit columns  
-  TODO_type: number + enum columns  
-  TODO_shin: just number (use celsuis)  
-  TODO_sanjay:
--
-  name: biospecimen_aliquot_temperature_in_preservation_solution_unit
+  name: specimen_preservation_temperature
   constraints:
     required: True
     enum:
-      - Celsius
-  description: Temperature unit
+      - -196 Celsius
+      - -80 Celsius
+      - -20 Celsius
+      - Room Temperature
+  description: The temperature of the medium during the preservation process.
 -
-  name: biospecimen_selection_qa_qc_measure_criteria_institution  
+  name: specimen_quality_criteria  
   constraints:
-    required: False
-  enum: {source_site,receiving_assay_site}  
-  description: "For example, RIN: 8.7. TODO - What is the source_site,receiving_assay_site? Two fields? Any format constraint possible?"
-
-  TODO_summation: optional  
-  TODO_nils: We don't know how this is supposed to be used. Should people provide either ""source_site"" or ""receiving_assay_site""?   
-  TODO_type:   
-  TODO_shin: "free text.  Could be 'RIN: 8.7'"  
-  TODO_sanjay:
+    required: False  
+  description: 'For example, RIN: 8.7.'
 -
   name: biospecimen_selection_distance_from_tumor_value 
```

Comment:
Ok, making it an optional numeric field: In discussion, there was the idea that it could be a number, or "Unknown", but it that case I guess it should be left blank.